frontends/web/src/api/account.ts

```diff
@@ -324,3 +324,29 @@ export const addAccount = (coinCode: string, name: string): Promise<TAddAccount>
 export const testRegister = (pin: string): Promise<null> => {
   return apiPost('test/register', { pin });
 };
+
+export type TSignMessage = { success: false, aborted?: boolean; errorMessage?: string; } | { success: true; signature?: string; }
```

Comment:
should the signature not be mandatory in case of success, i.e. `signature: string` (without the question mark)?